package.json

```diff
@@ -76,7 +76,7 @@
     "verify-fnm-package": "node ./.ci/prepare-fnm-package.js --fail-on-difference",
     "bootstrap": ".ci/bootstrap",
     "test": "esy x TestFnm.exe",
-    "fmt": "bash -c 'refmt --in-place {library,executable,test}/*.re'",
+    "fmt": "bash -c 'esy @fmt refmt --in-place ./*/*.re'",
```

Comment:
Do I need to do something to get this to work? Currently I get:

> error: project is not installed, run `esy install`

Is `@fmt` a project, and if so, how do I install it?